tools/ci_build/build.py

```diff
@@ -1310,6 +1310,9 @@ def generate_build_tree(
     if args.use_azure:
         add_default_definition(cmake_extra_defines, "onnxruntime_USE_AZURE", "ON")
 
+    if is_windows():
+        add_default_definition(cmake_extra_defines, "CMake_MSVC_PARALLEL", "ON")
```

Comment:
is this option for building CMake? was trying to find documentation for it and found this:
https://gitlab.kitware.com/cmake/cmake/-/issues/17696

Reply:
Nice catch! I misunderstood it.